phpstan.neon

```diff
@@ -1,7 +1,8 @@
 includes:
     - phpstan-baseline.neon
 parameters:
+    level: 7
     inferPrivatePropertyTypeFromConstructor: true
+    checkMissingIterableValueType: false
```

Comment:
If possible I'd prefer to keep this `true`

Reply:
I'll fix it.

Reply:
Interestingly enough, these are the reported errors:
```
------ ----------------------------------------------------------------------------------------------------------------------------------------------------------------- 
  Line   WorsePlainTextClassDefinitionLocator.php                                                                                                                         
 ------ ----------------------------------------------------------------------------------------------------------------------------------------------------------------- 
  27     Property Phpactor\WorseReferenceFinder\WorsePlainTextClassDefinitionLocator::$breakingChars type has no value type specified in iterable type array.             
         💡 Consider adding something like array<Foo> to the PHPDoc.                                                                                                       
         You can turn off this check by setting checkMissingIterableValueType: false in your phpstan.neon.                                                                
  34     Method Phpactor\WorseReferenceFinder\WorsePlainTextClassDefinitionLocator::__construct() has parameter $breakingChars with no value type specified in iterable   
         type array.                                                                                                                                                      
         💡 Consider adding something like array<Foo> to the PHPDoc.                                                                                                       
         You can turn off this check by setting checkMissingIterableValueType: false in your phpstan.neon.                                                                
  143    Method Phpactor\WorseReferenceFinder\WorsePlainTextClassDefinitionLocator::resolveImportTable() return type has no value type specified in iterable type array.  
         💡 Consider adding something like array<Foo> to the PHPDoc.                                                                                                       
         You can turn off this check by setting checkMissingIterableValueType: false in your phpstan.neon.                                                                
 ------ ----------------------------------------------------------------------------------------------------------------------------------------------------------------- 
```
Not sure why _I_ changed this setting...

Reply:
I'd appreciate suggestions. My laptop will be in the repair shop tomorrow, but after that I can make changes.

Reply:
I added phpstan at level 7 to all repos recently and generated a baseline for all of them, maybe this was created before that change. I think if you rebase they will go away, otherwise I think there are only a few failures as above so should be easy to eradicate them and remove the baseline.

Reply:
I'll try rebasing

Reply:
We now have several failing tests. Can you take a look?

Reply:
hmm looks good to me?

Reply:
composer integrate failed on my machine. Probably out of date dependencies?

Reply:
If you point me to which I should update, it would save me a ton of time tomorrow.

Reply:
what are the failures?

Reply:
also try clearing the result cache `phpstan clear-result-cache`

Reply:
```
1) Phpactor\WorseReferenceFinder\Tests\Unit\WorsePlainTextDefinitionLocatorTest::testGotoWord with data set "xml attribute" ('<element class="Foob<>ar">', 'Foobar.php')
Phpactor\ReferenceFinder\Exception\CouldNotLocateDefinition: Word ""Foobar"" could not be resolved to a class

/root/phpactor/worse-reference-finder/lib/WorsePlainTextClassDefinitionLocator.php:56
/root/phpactor/worse-reference-finder/tests/DefinitionLocatorTestCase.php:21
/root/phpactor/worse-reference-finder/tests/Unit/WorsePlainTextDefinitionLocatorTest.php:18

Caused by
Phpactor\WorseReflection\Core\Exception\SourceNotFound: Could not find source with ""Foobar""

/root/phpactor/worse-reflection/lib/Core/SourceCodeLocator/ChainSourceLocator.php:58
/root/phpactor/worse-reflection/lib/Core/Reflector/CoreReflector.php:126
/root/phpactor/worse-reflection/lib/Core/Reflector/CompositeReflector.php:73
/root/phpactor/worse-reference-finder/lib/WorsePlainTextClassDefinitionLocator.php:54
/root/phpactor/worse-reference-finder/tests/DefinitionLocatorTestCase.php:21
/root/phpactor/worse-reference-finder/tests/Unit/WorsePlainTextDefinitionLocatorTest.php:18

Caused by
Phpactor\WorseReflection\Core\Exception\SourceNotFound: Could not find source for ""Foobar"" in stub directory "/root/phpactor/worse-reference-finder/tests/Workspace"

/root/phpactor/worse-reflection/lib/Core/SourceCodeLocator/StubSourceLocator.php:43
/root/phpactor/worse-reflection/lib/Core/SourceCodeLocator/ChainSourceLocator.php:42
/root/phpactor/worse-reflection/lib/Core/Reflector/CoreReflector.php:126
/root/phpactor/worse-reflection/lib/Core/Reflector/CompositeReflector.php:73
/root/phpactor/worse-reference-finder/lib/WorsePlainTextClassDefinitionLocator.php:54
/root/phpactor/worse-reference-finder/tests/DefinitionLocatorTestCase.php:21
/root/phpactor/worse-reference-finder/tests/Unit/WorsePlainTextDefinitionLocatorTest.php:18
```

Reply:
I updated everything. It's gone.

Reply:
ah. it required the latest `phpactor/text-document`